src/Core/src/Platform/iOS/WrapperView.cs

```diff
@@ -73,7 +73,7 @@ public override void LayoutSubviews()
 				ShadowLayer.Frame = Bounds;
 
 			if (BorderView != null)
-				BringSubviewToFront(BorderView);
+				BorderView.Frame = Bounds;
```

Comment:
We no longer need to bring it to front? @jsuarezruiz?

Reply:
Tried and is not necessary. Is already done in other place.